src/main/webapp/app/exercises/quiz/manage/quiz-exercise-detail.component.ts

```diff
@@ -239,9 +239,21 @@ export class QuizExerciseDetailComponent extends QuizExerciseValidationDirective
         this.cacheValidation();
     }
 
+    /**
+     * Validates if the date is correct
+     */
+    validateDate() {
+        this.exerciseService.validateDate(this.quizExercise);
```

Comment:
Added a field `initCompleted` to prevent running the `validateDate` during the first call inside `init()`. Now it is not triggered immediately when the page is loaded so back button works without any warning.

Reply:
this doesn't actually properly resolve the issue as making a change and undoing it still has the issue, but the current state is probably good enough for now, but i would add a TODO comment for a future revisit with the additional context that quiz exercises are for some reason the only exercise type the has the unsaved changes warning.